diff --git a/src/confetti.js b/src/confetti.js
index 8fdbea2..modifié 100644
--- a/src/confetti.js
+++ b/src/confetti.js
@@ -246,6 +246,7 @@
     scalar: 1,
     rotate: false,
     rotationSpeed: null,
+    noFade: false
   };

@@ function randomPhysics(opts) {
     return {
       ...
       flat: opts.flat,
       rotate: opts.rotate,
+      noFade: opts.noFade
     };
   }

@@ function updateFetti(context, fetti) {
     ...
-    var progress = (fetti.tick++) / fetti.totalTicks;
+    var progress = (fetti.tick++) / fetti.totalTicks;
+    var alpha = fetti.noFade ? 1 : (1 - progress);

     ...
-    context.fillStyle = 'rgba(' + fetti.color.r + ', ' + fetti.color.g + ', ' + fetti.color.b + ', ' + (1 - progress) + ')';
+    context.fillStyle = 'rgba(' + fetti.color.r + ', ' + fetti.color.g + ', ' + fetti.color.b + ', ' + alpha + ')';

@@ if (fetti.shape.type === 'bitmap') {
-    context.globalAlpha = (1 - progress);
+    context.globalAlpha = alpha;
     ...
@@ function fireLocal(options, size, done) {
     ...
     var flat = !!prop(options, 'flat');
     var rotate = !!prop(options, 'rotate');
     var rotationSpeed = prop(options, 'rotationSpeed', Number);
+    var noFade = !!prop(options, 'noFade');

     ...
             scalar: scalar,
             flat: flat,
             rotate: rotate,
             rotationSpeed: rotationSpeed,
+            noFade: noFade
